docs/zh/quickstart/openmldb_quickstart.md

```diff
@@ -27,16 +27,26 @@ docker run -it 4pdosc/openmldb:0.4.4 bash
 :::
 
 ````{note}
-本教程将在单个容器中启动 OpenMLDB 单机版或集群版，如果希望从外部访问该容器内的 OpenMLDB 服务端口，启动容器时需要 `-p` 暴露端口。
+本教程将在单个容器中启动 OpenMLDB 单机版或集群版。
+请注意，OpenMLDB 所有 server 默认配置的 endpoint 地址都是 `127.0.0.1`。
+如果希望从本机访问**本机容器内**的 OpenMLDB 服务端口，需要将回环地址改为 `0.0.0.0`，并且在启动容器时需要 `-p` 暴露端口。
+
+单机版：
 ```
 docker run -p 6527:6527 -p 2181:2181 -p 8080:8080 -it 4pdosc/openmldb:0.4.4 bash
 ```
+
+集群版：
+```
+docker run -p 7527:7527 -p 10921:10921 -p 10922:10922 -p 2181:2181 -p 8080:8080 -it 4pdosc/openmldb:0.4.4 bash
+```
+
 或者更方便地，使用 host networking，不进行端口隔离，例如
 ```
 docker run --network host -it 4pdosc/openmldb:0.4.4 bash
 ```
-注意，OpenMLDB 所有 server 配置的 endpoint 地址都是 `127.0.0.1` 。如果想要外部主机访问本机容器内的服务，需要将回环地址改为外网 IP 或 `0.0.0.0` 。
 
+TODO: 外部主机绑定端口的docker启动方式，与注意事项
```

Comment:
this explanation for this TODO is not done yet?

Reply:
> i feel like the description about connecting OpenMLDB from outside of the container is a bit complicated for a quickstart, i am thinking about moving all those relevant explanations to another doc (such as a README for the docker image)

Agree.
And about the TODO, I don't have time to test, but the port in cmd is wrong, so emergency repair.

Reply:
ok, let's fix the port first then, and we may move those contents to another place later on